### ISSUE TITLE
YMMB Maneuvering Area Responsibility

### ISSUE THREAD

**Issue by steve-gehle** [vatpac-technology/sops#410]
The [YMMB SOP](https://sops.vatpac.org/aerodromes/classd/Moorabbin/) doesn't specify maneuvering area responsibility. This can cause confusion when SMC and ADC are online. Providing a link to, replicating or paraphrasing [the CASA reference](https://www.casa.gov.au/moorabbin-manoeuvring-area-map), like [this](https://sops.vatpac.org/aerodromes/classd/Moorabbin/#arrivalsoverfliers) for YSSY, will help clarify what SMC and ADC cover.

I'm happy to contribute this change via PR, please let me know if OK to proceed.

![image](https://github.com/user-attachments/assets/18ca837a-3a5d-4b26-9611-e47191f16650)

**Comment by steve-gehle** [comment]
> Thanks for the suggestion @steve-gehle, feel free to submit this as a PR and we can review from there. Thanks for your input!

Is there a preference for diagram style? i.e., to link external sources, use existing templates from other ADs for maneuvering area responsibility, or author our own based on vatSys?

**Comment by alphadelta332** [comment]
> > Thanks for the suggestion @steve-gehle, feel free to submit this as a PR and we can review from there. Thanks for your input!
> 
> Is there a preference for diagram style? i.e., to link external sources, use existing templates from other ADs for maneuvering area responsibility, or author our own based on vatSys?

Yeah we generally create our own using vatsys, to avoid any copyright infringements. 

[airport diagrams.zip](https://github.com/user-attachments/files/18190544/airport.diagrams.zip)

I've attached some exisiting PSDs we've used, so you can base your formatting on that. Cheers!

**Comment by steve-gehle** [comment]
Submitted PR to allow others to continue work on this, without my availability being a potential blocker.

**Comment by m-9c** [comment]
I'm noticing that there is inconsistency with how maneuvering areas are document between aerodromes.

For airports such as Sydney, the diagram shows areas of responsibility, but does not distinguish between maneuvering/apron areas.

For others, such as Alice Springs, only the maneuvering areas are displayed. The aprons are unshaded.

Probably need to establish a convention on how/what is drawn, and potentially adjust existing diagrams to match.
